packages/shared/lib/contexts/governance/interfaces/proposal-state.interface.ts

```diff
@@ -0,0 +1,7 @@
+import type { EventStatus } from '@iota/wallet'
+
+export interface IProposalState {
```

Comment:
Probably should be named `IProposalsState` to be consistent with the wallet-rs naming and also the `proposalsState` store.